dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -248,35 +296,15 @@ def test_profile(self):
         with utils.mock_timeit():
             profile = graph_profile.update(self.graph_1)
 
-        # check that scale is almost equal
-        scale = profile.profile["continuous_distribution"]["weight"].pop("scale")
-        self.assertAlmostEqual(scale, -15.250985118262854)
-
-        # check that properties are almost equal
-        properties = profile.profile["continuous_distribution"]["weight"].pop(
-            "properties"
-        )
-        self.check_continuous_properties(properties)
-
-        self.assertDictEqual(self.expected_profile, profile.profile)
+        self.check_profile(profile)
```

Comment:
This should be all you need here:
```suggestion
        # check that properties are almost equal
        weight_properties = profile.profile["continuous_distribution"]["weight"].pop(
            "properties"
        )
        uniform_properties = profile.profile["continuous_distribution"]["uniform"].pop(
            "properties"
        )
        
        self.check_continuous_properties(weight_properties, self.expected_properties_weight)
        self.check_continuous_properties(uniform_properties, self.expected_properties_uniform)
        self.assertDictEqual(self.expected_profile, profile.profile)
```